readme.md

```diff
@@ -76,11 +77,15 @@
 - https://www.buybitcoinworldwide.com/fee-calculator/
 - http://charts.woobull.com
 
+## Bloqboard
+- https://app.bloqboard.com
+
```

Comment:
lets put this in the loans section, since its an aggregator?

Reply:
Yes, that makes sense